scripts/update_packages.ts

```diff
@@ -0,0 +1,162 @@
+  'react-i18next': '11.8.15',
+  'react-infinite-scroll-component': '5.0.4',
+  'react-intersection-observer': '8.31.0',
+  'react-is': '16.13.1',
+  'react-markdown': '6.0.3',
+  'react-moment': '1.1.1',
+  'react-player': '2.9.0',
+  'react-query': '3.5.12',
+  'react-redux': '7.1.1',
+  'react-router': '5.2.1',
+  'react-router-dom': '5.3.0',
+  'react-sortable-hoc': '1.11.0',
+  redux: '4.0.1',
+  'redux-devtools-extension': '2.13.8',
+  'redux-mock-store': '1.5.3',
+  'redux-saga': '1.1.3',
+  'redux-thunk': '^2.4.1',
+  'styled-components': '5.3.1',
+  typescript: '4.5.2',
+  enzyme: '3.10.0',
+  'enzyme-adapter-react-16': '1.15.1',
+  eslint: '7.32.0',
+  'eslint-config-prettier': '8.3.0',
+  'eslint-config-standard': '16.0.3',
+  'eslint-plugin-cypress': '2.12.1',
+  'eslint-plugin-header': '3.1.1',
+  'eslint-plugin-i18next': '5.1.2',
+  'eslint-plugin-import': '2.25.3',
+  'eslint-plugin-jest': '25.3.0',
+  'eslint-plugin-jest-dom': '4.0.0',
+  'eslint-plugin-lodash': '7.3.0',
+  'eslint-plugin-mdx': '1.16.0',
+  'eslint-plugin-node': '11.1.0',
+  'eslint-plugin-prettier': '4.0.0',
+  'eslint-plugin-promise': '6.0.0',
+  'eslint-plugin-react': '7.27.0',
+  'eslint-plugin-react-hooks': '4.3.0',
+  'eslint-plugin-sort-keys-fix': '1.1.2',
+  'eslint-plugin-testing-library': '5.0.1',
+  'event-hooks-webpack-plugin': '2.2.0',
+  jest: '26.6.3',
+  'jest-axe': '4.1.0',
+  'jest-canvas-mock': '2.2.0',
+  'jest-codemods': '0.15.0',
+  'jest-environment-jsdom': '26.6.2',
+  'jest-fetch-mock': '2.1.2',
+  'jest-html-reporters': '1.2.1',
+  'jest-junit': '12.1.0',
+  'jest-styled-components': '7.0.3',
+  webpack: '5.67.0',
+  'webpack-bundle-analyzer': '4.5.0',
+  'webpack-cli': '4.9.1',
+  'webpack-merge': '5.8.0',
+  'webpack-subresource-integrity': '5.1.0',
+  prettier: '2.2.1',
+  'prettier-check': '^2.0.0',
+  '@styled-icons/material': '10.28.0',
+  '@styled-icons/material-outlined': '10.34.0',
+  '@styled-icons/material-rounded': '10.34.0',
+  '@babel/cli': '7.16.0',
+  '@babel/core': '7.16.0',
+  '@babel/eslint-parser': '7.16.5',
+  '@babel/plugin-transform-runtime': '7.16.4',
+  '@babel/preset-env': '7.16.4',
+  '@babel/preset-react': '7.16.0',
+  '@babel/preset-typescript': '7.16.0',
+  '@babel/runtime-corejs3': '7.16.3',
+  '@types/styled-components': '5.1.13',
+  '@types/styled-system': '5.1.13',
+}
+
+const isIgnorable = (fileName: string) => {
+  const ignoreDirs = ['node_modules', '.git', '..', '.']
+  return ignoreDirs.includes(fileName)
+}
+
+const findPackageFiles = (dirName: string, packageFiles: string[]) => {
+  const files = fs.readdirSync(dirName)
+  files.forEach((fileName) => {
+    if (!isIgnorable(fileName)) {
+      const name = `${dirName}/${fileName}`
+      if (fs.lstatSync(name).isDirectory()) {
+        findPackageFiles(name, packageFiles)
+      } else if (fileName === 'package.json') {
+        packageFiles.push(name)
+      }
+    }
+  })
+}
+
+const updateContent = (lines: string[]) => {
```

Comment:
Do you think using a JSON parsed object could make this replacement logic more robust?

Reply:
I did think about that but my concern was the potential side effect of changes that happen when written out. I would have to reply on some version of JSON.stringify and wasnt sure what the changes would be. With this albeit ugly change I know the only thing that will change is the dependencies.

Reply:
It's working, so 💯  it's ready. I think it could have been safe to JSON.parse for processing keys then `JSON.stringify(content, null, 2)` but you never know if prettier will see it the same way.